### PR TITLE
Update for AERI v2.0.4 and regenerate .targets

### PR DESCRIPTION
**Pull request by briandealwis**

### CODE REVIEW

File: features/com.google.cloud.tools.eclipse.suite.e45.feature/feature.xml [hunk 1]
```diff
@@ -29,7 +29,6 @@
       <import feature="org.eclipse.wst.web_ui.feature" version="3.7.1.v201602111638" match="greaterOrEqual"/>
    </requires>
 
-   <!-- appengine-plugins-core -->
```
Comment (chanseokoh) [inline features/com.google.cloud.tools.eclipse.suite.e45.feature/feature.xml]: I think it's okay to leave this comment here. I was confused once why we don't have this `com.google.cloud.tools.appengine` in my workbench.

File: eclipse/oxygen/gcp-eclipse-oxygen.target [hunk 2]
```diff
@@ -24,7 +24,6 @@
       <unit id="org.eclipse.jetty.servlet" version="9.4.0.v20161208"/>
       <unit id="org.eclipse.jetty.server" version="9.4.0.v20161208"/>
       <unit id="org.eclipse.jetty.util" version="9.4.0.v20161208"/>
-      <unit id="ch.qos.logback.slf4j" version="1.0.7.v201505121915"/>
```
Comment (chanseokoh) [inline eclipse/oxygen/gcp-eclipse-oxygen.target]: Is this related to this issue?

File: eclipse/mars/gcp-eclipse-mars.target [hunk 3]
```diff
@@ -25,8 +25,8 @@
       <repository location="http://download.eclipse.org/releases/mars/"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.eclipse.epp.logging.aeri.feature.feature.group" version="2.0.3.v20160923-1310"/>
-      <unit id="org.eclipse.epp.logging.aeri.feature.source.feature.group" version="2.0.3.v20160923-1310"/>
+      <unit id="org.eclipse.epp.logging.aeri.feature.feature.group" version="2.0.4.v20170307-1435"/>
+      <unit id="org.eclipse.epp.logging.aeri.feature.source.feature.group" version="2.0.4.v20170307-1435"/>
```
Comment (chanseokoh) [inline eclipse/mars/gcp-eclipse-mars.target]: You didn't change this for Neon or Oxygen. (They still have entries with "2.0.3".) Is this intended?

Comment (briandealwis) [inline eclipse/mars/gcp-eclipse-mars.target]: AERI is shipped as part of the Neon and Oxygen builds, and they'll be updated for the next milestone.  I think we want to build with the shipped versions.

File: eclipse/oxygen/gcp-eclipse-oxygen.target (hunk 2 above)
Comment (briandealwis) [inline eclipse/oxygen/gcp-eclipse-oxygen.target]: No, but this dependency shouldn't have been here as I previously moved Logback to be resolved from the Orbit repository (which you can see further below).

File: features/com.google.cloud.tools.eclipse.suite.e45.feature/feature.xml (hunk 1 above)
Comment (briandealwis) [inline features/com.google.cloud.tools.eclipse.suite.e45.feature/feature.xml]: I used the feature editor to remove the dependency; I'll restore it.
